models/netsuite2/intermediate/base/int_netsuite2__transaction_lines.sql

```diff
@@ -28,6 +65,11 @@ joined as (
     left join transaction_accounting_lines
         on transaction_lines.transaction_line_id = transaction_accounting_lines.transaction_line_id
         and transaction_lines.transaction_id = transaction_accounting_lines.transaction_id
```

Comment:
Hello @jmongerlyra thank you for your reply and the SQL code.
The case I mentioned last week is not returned with the query you shared. It seems it was the timing lag between these two tables ingested via the connector. Both tables now contain line_ids 0, 1, and 3. 👍 However I see the Fivetran sync timestamp being 
2023-12-08 **17:28:23.138000** UTC for one and 
2023-12-08 **23:26:57.051000** UTC for the other.

Beside that case that seem self resolved, the query returns **238 records**. What should I do with that?

Reply:
I'll let @fivetran-joemarkiewicz or someone else from Fivetran provide input. Seems like a timing issue and unrelated to this PR since the first example self corrected.

Reply:
Hi @erwann-pleo thank you for bringing this to our attention, and thanks @jmongerlyra for looking into this! Since this appears connector-related, I don't think there is much the package team can do at the moment, but I am bringing this to our internal team for discussion. I'll keep you posted on what they say about it.

Reply:
Hello @fivetran-catfritz! Agreed, this is about the connector, not the package. Thank you both